src/app/pages/checkout/checkout-page.module.ts

```diff
@@ -44,6 +45,7 @@ const checkoutPageRoutes: Routes = [
       },
       {
         path: 'receipt',
+        canActivate: [ServerSideRenderingGuard],
```

Comment:
Shouldn't this be attached to all `checkout` routes in the `app-routing.module`?

Reply:
I attached it to all checkout routes in the `checkout-page.module.ts`. This seemed the better place to me since one can see better the combination with the `CheckoutPageGuard` at this place for some routes.